client/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { ButtonStyles, LoginButtonStyles } from "./ButtonStyles";
+
+type Props = {
+  content: string;
+  name: string;
+  type: string;
+}
+
+export default function Button({ content, name }: Props) {
+  return <ButtonStyles id={name + "-button"}>{content}</ButtonStyles >;
```

Comment:
You can just replace `{content, name}: Props` with `props: Props` then you can refer to them on line 11 like this: `props.name` and `props.content`. This should make it more scalable in the future. You should also add `type={props.type}` to this `ButtonStyles` tag because you are passing it though as a prop in Search.tsx and Login.tsx.

Reply:
![image](https://user-images.githubusercontent.com/70618345/192123950-2cca259c-8e8f-4361-a293-96e4e4c6b635.png)


I'm currently running into this problem, apparently there is a prop called children being passed into the Button component, but it is not explicitly passed as you can see in the screenshot.